docs/08-clusterconnectionstring.md

```diff
@@ -1,7 +1,55 @@
 Cluster Connection String
 ===============
 
-## Create a user
+*Terraform users can skip to our [generate connection string section](#generate-connection-string) as Terraform will have created and set up your role and user for you.*
+
+---------------
+
+## Ensure Your Role has Correct Permissions (If manually set up (not useing Terraform))
+
+### Policy
+
+The policy you created need to have at least these permissions (Note the ARN should match that of your created cluster):
+
+    {
+        "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Action": "finspace:ConnectKxCluster",
+                    "Resource": "<ENVIRONMENT_ARN_COPIED_FROM_KDB_ENIRONMENT_PAGE>/kxCluster/*"
+                },
+                {
+                    "Effect": "Allow",
+                    "Action": "finspace:GetKxConnectionString",
+                    "Resource": "<ENVIRONMENT_ARN_COPIED_FROM_KDB_ENIRONMENT_PAGE>/kxCluster/*"
+                }
+        ]
+    }
+
```

Comment:
Might want to update this so it matches with permissions mentioned on the bottom half of this page : https://docs.aws.amazon.com/finspace/latest/userguide/permissions-managed-kdb.html

Reply:
Do you mean for the S3 bucket permissions? If so, this is something we should likely update in the MVP terraform too (as it will start to block once AWS does their release on this, if they haven't already).